snowpack.dev.config.cjs

```diff
@@ -8,6 +8,7 @@ module.exports = {
     'src/js': { url: '/dist' },
     // Mount "public" to the root URL path ("/*") and serve files with zero transformations:
     examples: { url: '/examples', static: true, resolve: false },
+    sandbox: { url: '/sandbox', static: true, resolve: false },
```

Comment:
? was this intentional?

Reply:
Yeah, intentional. Before switching to snowflake this just worked. I've always appreciated having a sandbox folder I can toss test files in and not worry about messing with actual project file like demos and examples. (i.e. [video.js/sandbox](https://github.com/videojs/video.js/tree/main/sandbox)) I should probably add a .example file too to make it clear what you can do with it.